Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -95,6 +99,30 @@ using MCParticlesTable = o2::soa::Table<o2::aod::mcparticle::McCollisionId,
                                         o2::aod::mcparticle::Vz,
                                         o2::aod::mcparticle::Vt>;
 
+typedef boost::tuple<int, int, int> triplet_t;
+
+struct TripletHash : std::unary_function<triplet_t, std::size_t> {
+  std::size_t operator()(triplet_t const& e) const
+  {
+    std::size_t seed = 0;
+    boost::hash_combine(seed, e.get<0>());
+    boost::hash_combine(seed, e.get<1>());
+    boost::hash_combine(seed, e.get<2>());
+    return seed;
+  }
+};
+
+struct TripletEqualTo : std::binary_function<triplet_t, triplet_t, bool> {
+  bool operator()(triplet_t const& x, triplet_t const& y) const
+  {
+    return (x.get<0>() == y.get<0>() &&
+            x.get<1>() == y.get<1>() &&
+            x.get<2>() == y.get<2>());
+  }
+};
+
+typedef boost::unordered_map<triplet_t, int, TripletHash, TripletEqualTo> tripletsMap_t;
```

Comment:
The PR looks ok to me. Just 2 things:
a) types should start with capitcal letters. So please use "Triplet_t" and "TripletsMap_t".
b) why boost? Is std::unordered_map not good enough?

Reply:
Hi, @sawenzel.
a): I have fixed types naming
b): There is no hash function implementation for `std::tuple`, so one need to implement it. Second, in order to use `unordered_map` with tuples, I would need to put my hash function and comparison operator inside `std` namespace (I may be wrong here). So I thought that it is more convenient to use `boost::tuple` and `boost::unordered_map`.